inception-imls-opennlp/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/opennlp/doccat/OpenNlpDoccatRecommender.java

```diff
@@ -180,7 +176,7 @@ public EvaluationResult evaluate(List<CAS> aCasses, DataSplitter aDataSplitter)
         
         if (trainingSetSize < 2 || testSetSize < 2) {
             String info = String.format(
-                    "Not enough evaluation data: training set [%s] items, test set [%s] of total [%s].",
+                    "Not enough training data: training set [%s] items, test set [%s] of total [%s].",
```

Comment:
Hm, no... why is this change necessary?

Reply:
This is an unnecessary change... I originally wanted to split the if statement, but I found that it didn't make sense.